ncm-ceph/src/main/pan/components/ceph/schema-rgw.pan

```diff
@@ -0,0 +1,30 @@
+declaration template components/${project.artifactId}/schema-rgw;
+
+@documentation{ ceph rados gateway config }
+type ceph_radosgw_config = { 
+    include ceph_daemon_config
+    'host'      : string
+    'keyring'   : string
+    'rgw_socket_path' : string = ''
+    'log_file'  : string = '/var/log/radosgw/client.radosgw.gateway.log'
+    'rgw_frontends' : string = "\"civetweb port=8000\""
```

Comment:
Hmm, i don't find this option documented somewhere, but I've found some examples where it is not quoted, so I guess It don't has to be this dirty. @gvasilak can I remove the inner quotes?